packages/playground/src/pages/OpenInvoices/OpenInvoices.js

```diff
@@ -56,9 +56,9 @@ getPaymentMethods({ amount, shopperLocale }).then(paymentMethodsData => {
         .create('affirm', {
             countryCode: 'US', // 'US' / 'CA'
             visibility: {
-                personalDetails: 'editable', // editable [default] / readOnly / hidden
-                billingAddress: 'editable',
-                deliveryAddress: 'editable'
+                personalDetails: 'hidden', // editable [default] / readOnly / hidden
+                billingAddress: 'hidden',
+                deliveryAddress: 'hidden'
```

Comment:
Hey @pabloai, are these changes intentional?

Reply:
They weren't, thanks for checking! I just pushed the rollback.